V2rayNG/app/src/main/kotlin/com/v2ray/ang/ui/ServerCustomConfigActivity.kt

```diff
@@ -91,7 +91,7 @@ class ServerCustomConfigActivity : BaseActivity() {
         }
 
         val config = MmkvManager.decodeServerConfig(editGuid) ?: ServerConfig.create(EConfigType.CUSTOM)
-        config.remarks = binding.etRemarks.text.toString().trim()
+        config.remarks = v2rayConfig.remarks ?: binding.etRemarks.text.toString().trim()
```

Comment:
Because this is manual editing, there is no need to do this.

Reply:
Its not manual edit, i set remarks from custom config's remarks field in its json if any, otherwise set timestamp for its name. This way config those have remark in related json will show by ther remark in the list, instead of a random timestamp.

Reply:
If you ignore this part, editting custom config's remark inside the app will have no effect on the config name in the list. Btw its not that important.